triton-vm/src/table/extension_table.rs

```diff
@@ -342,24 +343,52 @@ pub trait Quotientable: ExtensionTable + Evaluable {
         challenges: &AllChallenges,
         omicron: XFieldElement,
         padded_height: usize,
+        maybe_profiler: &mut Option<TritonProfiler>,
     ) -> Vec<Vec<XFieldElement>> {
+        if let Some(p) = maybe_profiler.as_mut() {
+            p.start("initial quotients")
+        }
```

Comment:
I recommend using the macros `prof_start` and `prof_stop`. Example use can be seen [here](https://github.com/TritonVM/triton-vm/blob/master/triton-vm/src/stark.rs#L147).